scipy/stats/tests/test_stats.py

```diff
@@ -6704,6 +6707,166 @@ def test_dist_perm(self):
         assert_approx_equal(pvalue_dist, 0.001, significant=1)
 
 
+class TestFNI:
+    @pytest.mark.parametrize(("distname", "shapes"), distcont)
+    def test_FastNumericalInverse(self, distname, shapes):
```

Comment:
As these tests are present in a class dedicated to FastNumericalInverse, I think it would be OK to remove the phrase "FastNumericalInverse" from the method names. Something like this would be better:

```suggestion
    def test_method(self, distname, shapes):
```